templates/error_page.hbs

```diff
@@ -10,12 +9,41 @@
     <h2>{{t 'not_authorized'}}</h2>
   {{/is}}
 
-  {{#is error 'not_found'}}
-    <h2>{{t 'nonexistent_page'}}</h2>
-    <p>{{t 'mistyped_address_or_moved_page'}}</p>
-  {{/is}}
-
-  {{#link 'help_center'}}
-    {{t 'back_to_homepage'}}
-  {{/link}}
+{{#is error 'not_found'}}
+  <h1 class="page-title compact-bottom">404</h1>
+  <h2 class="compact-top">Page not found</h2>
+  <p>The link you followed may be broken or we may have moved the page. If you cannot find what you are looking for, <a href="https://www.artstor.org/contact-us">contact us</a>.
+  <p>
+  	<a href="/" class="button">Back to Homepage</a>
```

Comment:
Let's think about what buttons are useful here - right now, these both go to the same page. Do we just need one button? or do we need one that goes to the support home and one that goes to the platform home? May be a good question for Bonnie and Kelly